obfuscated/jasonzhao6/submission-2.js

```diff
@@ -0,0 +1,11 @@
+// Copy paste into Chrome's v69 browser console.
+
+[[
+    'Jan','Feb','Mar','Apr','May','Jun',
+    'Jul','Aug','Sep','Oct','Nov','Dec'
+
+  ].slice(6,11).map(e=>e[0]),' ',
+  new Date().toISOString().slice(-1),
+  [History,Array,Object].map(e=>e.name[0])
+
+].flat().join('')
```

Comment:
I had to update `.flat()` to `.flatten()` to get this to work. Can you update the PR?

Reply:
I see the [MDN doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat) for `.flat()`, but nothing for `.flatten()`. Weird that you got an error. I just asked Anthony to try, and it worked in his Chrome console. I'll stop by to take a peek on your machine today if you have a free moment.